package.json

```diff
@@ -12,9 +12,9 @@
   },
   "dependencies": {
     "markdown-it-mathjax3": "^4.3.2",
-    "markdown-it-vuepress-code-snippet-enhanced": "^1.0.1",
+    "markdown-it-vuepress-code-snippet-enhanced": "github:IMB11/md-it-enhanced-snippets",
```

Comment:
This needs to be a specific commit/version. Any changes to your repo will be automatically picked up here when we dont want them.

Reply:
```suggestion
    "markdown-it-vuepress-code-snippet-enhanced": "github:IMB11/md-it-enhanced-snippets#dfb9fa2"
```